src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"beige", "magenta", "blue", "yellow", "pink", "purple"};
+    private static final Random RANDOM = new Random();
+
+    public static String getRandomColor() {
```

Comment:
generally using static is a bad practice
```suggestion
    public String getRandomColor() {
```